src/Template/Element/Form/relations.twig

```diff
@@ -2,10 +2,12 @@
 <section class="relations">
 
     {% for relationName, relationLabel in relations %}
+        {% set acceptedTypes = uploadable | filter((item) => item in relationsSchema[relationName].right) | join(', ') %}
```

Comment:
in this context we could probably use something like `uploadableTypes` and `uploadableNum` (number of uploadable types in related types)